jdaviz/core/template_mixin.py

```diff
@@ -4482,9 +4620,13 @@ def clear_marks(self, *mark_labels):
                     else:
                         mark.x, mark.y = [], []
 
+        self._plugin.session.hub.broadcast(PluginPlotModifiedMessage(sender=self))
+
```

Comment:
I don't think anything is changed when just accessing the marks

```suggestion
```

Reply:
i thought clearing marks would count as modifying the plot?

Reply:
ah, I misread the diff and thought it was in the marks property.  Yes, let's keep this one gut remove the one in `clear_all_marks` then (which calls this)